### PR TITLE
fix: led strip

### PR DESCRIPTION
**Pull request by boczekbartek**

### CODE REVIEW

File: src/led_strip/led_strip/led_strip_node.py [hunk 1]
```diff
@@ -61,22 +61,18 @@ def calculate_state(self) -> str:
...
+        if self.hmi_state == "processing":
+            self.led_state = "processing"
+        if self.tts_state == "playing" or self.tts_state == "processing":
+            self.led_state = "playing"
+        if self.asr_state == "recording":
+            self.led_state = "recording"
+
+        self.get_logger().info(f'{self.hmi_state=}, {self.asr_state=}, {self.tts_state=}, {self.led_state=}')
+        self.get_logger().info(f'LED state: {self.led_state}')
+        return self.led_state
```
Comment (jhanca-robotecai) [inline src/led_strip/led_strip/led_strip_node.py]: Is this return ever used? I do not see any changes in the code to consume the value.

**Review by jhanca-robotecai — COMMENTED** [review]
The previous code was _robot-oriented_ - it was the robot waiting for an external signal to change the state of LEDs. Now the change is happening outside of the robot. The change is correct as long, as ASR, HMI and TTS are synchronized.

Btw. the previous code was supporting:
- ASR: `dropping`, recording, `transcribing`
- TTS: playing, `waiting`

the new one supports:
- ASR: recording, `waiting`
- HMI: `processing`
- TTS: playing, `processing`

Differences highlighted.

File: src/led_strip/led_strip/led_strip_node.py (hunk 1 above)
Comment (boczekbartek) [reply src/led_strip/led_strip/led_strip_node.py]: Indeed, it was never used. I removed it in: https://github.com/RobotecAI/rai-rosbot-xl-demo/pull/17/commits/632cc6cbf316e0c606a7304e4ea29b3f340ba83a

**Comment by boczekbartek** [comment]
@jhanca-robotecai 
Previously leds were not working correctly.

You are right, the current version depends on ASR, HMI and TTS being synchronized. It was tested on the robot extensively and is stable.

**Review by jhanca-robotecai — APPROVED** [review]
Thank you for the clarification, approve

**Pull request merged**